index.css

```diff
@@ -0,0 +1,547 @@
+  width: 95%;
+  float: right;
+  padding-left: 20px;
+}
+
+.comments-desc {
+  margin: 0;
+  display: inline-block;
+  width: 75%;
+}
+
+.home {
+  display: flex;
+}
+
+.footer-right-column-left {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  width: 40%;
+  height: auto;
+  margin: 10px;
+}
+
+.footer-right-column-right {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 40%;
+  height: auto;
+  margin: 10px;
+}
+
+.home-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.home-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+.phone-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.phone-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+.email-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.email-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+.twitter-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.twitter-desc {
+  /*background-color: maroon;*/
+  display: inline-block;
+  width: 90%;
+}
+
+.insta-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.insta-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+.dribbble-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.dribbble-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+.facebook-img {
+  background-color: mediumblue;
+  width: 20px;
+  height: 20px;
+  display: inline-flex;
+}
+
+.facebook-desc {
+  display: inline-block;
+  width: 90%;
+}
+
+#header2 {
+  border-bottom: 6px double grey;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 270px;
+  margin-right: 270px;
+}
+
+#header2 h1 {
+  font-size: xx-large;
+}
+
+#header2 h6 {
+  text-align: center;
+}
+
+h6 {
+  text-align: left;
+}
+
+#main2 {
+  display: flex;
+  flex-direction: row;
+  width: 65%;
+  padding-left: 200px;
+  justify-content: space-between;
+}
+
+.main-content2-wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 59%;
+  padding-right: 10px;
+  justify-content: flex-start;
+  align-content: flex-start;
+}
+
+.main-content2 {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+}
+
+.main-content2-sidebar {
+  width: 700px;
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+  margin-right: auto;
+}
+
+.main-content2-img1 {
+  width: 100%;
+}
+
+/*Page 3*/
+
+.main-content3-wrapper {
+  width: 55%;
+}
+
+.main-content3 p {
+  width: 530px;
+}
+
+.main-content3 {
+  padding-right: 80px;
+  padding-left: 20px;
+  width: 100%;
+}
+
+.main-content3-img1 {
+  width: 520px;
+  height: 200px;
+}
+
+.main-content-blog-sidebar {
+  width: 42%;
+  padding-left: 70px;
+  padding-right: 5px;
+}
+
+/*Page 4*/
```

Comment:
This type of naming convention makes sense when you're the one writing it. If you used the title of the page it would be more obvious what page this goes to for other people reading it

Reply:
Copy that.